xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-api/src/main/java/org/xwiki/realtime/RealtimeEditorManager.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.realtime;
+
+import org.xwiki.component.annotation.Role;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.stability.Unstable;
+
+/**
+ * Allows to probe for active realtime sessions.
+ * 
+ * @version $Id$
+ * @since 16.2.0
+ * @since 16.3.0RC1
+ */
+@Unstable
+@Role
+public interface RealtimeEditorManager
+{
+
+    /**
+     * Determine the currently selected editor.
+     * Expected outputs can be but are not limited to "inplace", "inline", "wysiwyg", "wiki"...
+     * 
+     * @return the currently selected editor.
+     */
+    String getSelectedEditor();
+
+    /**
+     * Determine if there is an active realtime session for the given document translation and edit mode.
+     * 
+     * @param target Specifies which document to check.
+     * @param locale Specifies the translation of the document to be selected.
+     * @param editor Specifies with which editor the realtime session is going to run.
+     * @return true if there is an active session, false otherwise.
+     */
+    Boolean sessionIsActive(DocumentReference target, String locale, String editor);
```

Comment:
Any reason for having a Boolean instead of a boolean? Instinctively when using Boolean I'm thinking it can be nullable but doesn't seem to be the case here.

Reply:
Also why not using a Locale type for the locale?